### PR TITLE
Add cirrus support

### PR DESCRIPTION
**Pull request by baude**
Initial PR to add cirrus support.

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by baude** [comment]
DO NOT MERGE YET

File: .cirrus.yml [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+    container:
+        cpu: 2
+        memory: 2
+        image: quay.io/libpod/imgts:$IMAGE_SUFFIX
+    env:
+        # Space-separated list of images used by this repository state
+        IMGNAMES: "${FEDORA_NETAVARK_IMAGE}"
+        BUILDID: "${CIRRUS_BUILD_ID}"
+        REPOREF: "${CIRRUS_REPO_NAME}"
+        GCPJSON: ENCRYPTED[4c8f37db84c8afb3d67932ebbf1f062e5e7e54b64e9f99624d96d828d2b8677624fb1470a9b12c097e06afeb11fb8c4e]
```
Comment (cevich) [inline .cirrus.yml:208]:
```suggestion
        GCPJSON: ENCRYPTED[c89506273e54e772a7899b9611e38ec72460855c4d1ad590cf0322b5d15bcb851859dfa0e39f962f991d5ddbdcf39ae0]
```

**Comment by baude** [comment]
Ok, ready for merge.

**Comment by mheon** [comment]
LGTM, pushing the button

**Pull request merged**